console/src/components/Services/ApiExplorer/GraphiQLWrapper/GraphiQLWrapper.js

```diff
@@ -50,16 +57,101 @@ class GraphiQLWrapper extends Component {
       graphqlNetworkData.headers
     );
 
+    let graphiqlContext;
+
+    // handle prettify
+    const handleClickPrettifyButton = () => {
+      const editor = graphiqlContext.getQueryEditor();
+      const currentText = editor.getValue();
+      const { parse, print } = require('graphql');
+      const prettyText = print(parse(currentText));
+      editor.setValue(prettyText);
+    };
+
+    // handle copy
+    const handleCopyQuery = () => {
+      const editor = graphiqlContext.getQueryEditor();
+      const query = editor.getValue();
+
+      if (!query) {
+        return;
+      }
+      copyToClipboard(query);
+      this.setState({ copyButtonText: 'Copied' });
+      setTimeout(
+        () => { this.setState({ copyButtonText: 'Copy' }); },
+        1500
+      );
+    };
+
+    // handle history toggle
```

Comment:
I would get rid of the comments here, since functions names are pretty self-explanatory.